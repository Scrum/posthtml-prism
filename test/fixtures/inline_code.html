The <code><div></code> is the generic container for flow content.