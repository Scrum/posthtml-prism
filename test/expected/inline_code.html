The <code><span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>div</span><span class="token punctuation">></span></span></code> is the generic container for flow content.